kitchen-ec2.gemspec

```diff
@@ -18,8 +18,11 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'test-kitchen', '~> 1.0'
-  gem.add_dependency 'fog'
+  gem.add_dependency "test-kitchen", "= 1.4.0.rc.1"
+  gem.add_dependency 'excon'
+  gem.add_dependency 'multi_json'
```

Comment:
When getting rid of Fog, I literally copied the transitive dependencies it was using (for the `fetch_credentials` call).  I should probably update that to just use json.